packages/botonic-nlp/src/dataset/entities-parser.ts

```diff
@@ -10,7 +10,8 @@ export class EntitiesParser {
 
     let text = sentence
     let m
-    let accumulation = 0
+
+    let charactersRemoved = 0 //This value represents the amount of characters removed because of entity definition removal: [shirt](product) -> shirt.
```

Comment:
Nobody likes reading long text/emails/comments. Be short
"This value represents the" adds no meaning. Always start comments directly with st like "amount of characters..."